src/components/ATATDialogBase.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <v-dialog
+    :max-width="width"
+    v-model="_showDialog"
+    role="alertdialog"
+    aria-labelledby="modalDialogTitle"
+    aria-describedby="modalDialogMessage"
+  >
+    <v-card>
+      <v-card-title class="h2 text-break" id="modalDialogTitle" tabindex="-1">
+        {{ getTitle }}
+      </v-card-title>
+      <v-card-text class="body-lg black--text mb-7" id="modalDialogMessage">
+        <slot name="content"></slot>
+      </v-card-text>
+      <v-card-actions class="d-flex justify-end">
+        <v-btn
+          class="link-button no-focus-shift"
+          :ripple="false"
+          @click="cancelItem"
+          id="dialog_cancel"
+          name="cancelDialog"
+          >{{ cancelText }}
+        </v-btn>
+        <v-btn
+          color="primary"
+          class="px-5"
+          :ripple="false"
+          id="dialog_ok"
+          @click="onOK"
+        >
+          {{ okText }}
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+import { Component, PropSync, Prop, Watch } from "vue-property-decorator";
+import { VDialog } from "vuetify/lib";
+import Vue from "vue";
+
+@Component({})
+export default class ATATModalDelete extends Vue {
+  @Prop() private message!: string;
+  @Prop({ default: "Dialog-Title" }) private title!: string;
+  @Prop() private id!: string;
+  @Prop({ default: "500px" }) private width!: string;
+  @Prop({ default: "Cancel" }) private cancelText!: string;
+  @Prop({ default: "OK" }) private okText!: string;
+  @Prop() private focusOnCancel!: string;
+  @Prop() private focusOnOk!: string;
+
+  @PropSync("showDialog")
+  private _showDialog!: boolean;
+
+  @Watch("showDialog")
+  setFocus(newVal: boolean): void {
+    if (newVal) {
+      this.$nextTick(() => {
+        setTimeout(function () {
+          document.getElementById("modalDeleteTitle")?.focus();
+        }, 100);
+      });
+    }
+  }
+
+  get getTitle(): string {
+    if (this.title && this.title.length > 60) {
+      return this.title.substring(0, 60) + "...”?";
+    }
+    return this.title;
+  }
+
+  private cancelItem() {
+    this._showDialog = false;
+    this.returnFocus(this.focusOnCancel);
+  }

```

Comment:
- [ ] when selecting `Cancel` with the tab/enter key, the user is to be returned to the button that activated the dialog.